lib/Elastica/Transport/NullTransport.php

```diff
@@ -26,9 +27,7 @@ class NullTransport extends AbstractTransport
     /**
      * Set response object the transport returns.
      *
-     * @param \Elastica\Response $response
-     *
-     * @return $this
+     * @return Response
```

Comment:
Missing `: Response` return type

Reply:
By default, it trying to return null, but nullable typehints allowed only in php 7.1

Reply:
Can't you use the method `generateDefaultResponse` in order to get a Response object?

Reply:
Sure